modules/custom/d_content/config/install/user.role.d_editor.yml

```diff
@@ -0,0 +1,50 @@
+uuid: 376e54d1-6692-4f64-966a-314c7ccb08d9
```

Comment:
hm, nie mam pewności czy to UUID strony czy roli, ale jeśli nie ma do niego odniesień (ctrl-shift-f w projekcie) to lepiej wywalic